bigtable/bigtable-proxy/src/main/java/com/google/cloud/bigtable/examples/proxy/channelpool/DataChannel.java

```diff
@@ -208,9 +263,13 @@ public void enterIdle() {
   @Override
   public <RequestT, ResponseT> ClientCall<RequestT, ResponseT> newCall(
       MethodDescriptor<RequestT, ResponseT> methodDescriptor, CallOptions callOptions) {
-    Optional.ofNullable(Tracer.extractTracerFromCallOptions(callOptions))
-        .map(Tracer::getCallLabels)
-        .ifPresent(resourceCollector::collect);
+    Tracer tracer =
+        Optional.ofNullable(Tracer.extractTracerFromCallOptions(callOptions))
```

Comment:
when will this happen?

Reply:
If we introduce a bug a to the call site and forget to embed the tracer in the call options